### PR TITLE
Crterai/addition to v2candidate ngd conv

### PR DESCRIPTION
**Pull request by crterai**
Updating the original run script for compy and adding a new run script for running on cori-knl.
Updates to original run script:
* update output variables and frequencies
* turn off state_debug_checks for normal simulations
* specify branch name

### CODE REVIEW

File: run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh [hunk 1]
```diff
@@ -0,0 +1,1622 @@
...
+ nucleate_ice_subgrid = 1.2d0
+ use_hetfrz_classnuc = .true.
+ do_Cooper_inP3 = .false.
+
+EOF
+
+cat >> user_nl_elm <<EOF
+
+ check_finidat_year_consistency = .true.
+ finidat = '/compyfs/linw288/inputdata/20201103.IELM.r05_oECv3.elm.r.0030-01-01-00000.nc'
```
Comment (JS-WRF-SBM) [inline run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh:1111]: @crterai 
Here I guess you want to use the file on Cori (please check that it's accessible for reading):
`/global/cfs/cdirs/e3sm/shpundk/elm_inputdata/20201103.IELM.r05_oECv3.elm.r.0030-01-01-00000.nc`

File: run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh [hunk 2]
```diff
@@ -0,0 +1,1622 @@
...
+
+#===========================
+# SET THE PARTITION OF NODES
+#===========================
+
+if ( `lowercase $debug_queue` == true ) then
+  if ( $machine == quartz || $machine == syrah ) then
+    $xmlchange_exe --id JOB_QUEUE --val 'pdebug'
+  else if ($machine != sandiatoss3 && $machine != bebop && $machine != blues) then
+    $xmlchange_exe --id JOB_QUEUE --val 'short'
```
Comment (JS-WRF-SBM) [inline run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh:906]: @crterai 
Here I think the `short` queue on Cori is named 'debug'

Comment (crterai) [reply run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh]: agreed. I've updated this now

File: run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh (hunk 1 above)
Comment (crterai) [reply run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh]: I was able to read this so. It has been changed

**Pull request merged**